client/src/CharacterSlider.js

```diff
@@ -32,39 +32,36 @@ const CharacterSlider = ({ maxval, setCounter, counter }) => {
   const [run, setRun] = useState(false);
   const [sliderVal, setSliderVal] = useState(0);
 
-  let scale = 100;
-
   useInterval(
     () => {
-      if (sliderVal > maxval * scale) {
+      if (sliderVal > maxval) {
         setRun(false);
       }
 
       setSliderVal(sliderVal + 1);
-      setCounter(Math.floor(sliderVal / scale));
+      setCounter(Math.floor(sliderVal));
     },
     run ? 50 : null
   );
 
   useEffect(() => {
-    setMark(generate_markers(maxval, scale));
+    setMark(generate_markers(maxval));
   }, [maxval]);
 
   useEffect(() => {
-    setSliderVal(counter * scale);
+    setSliderVal(counter);
   }, [counter]);
 
   return (
     <div>
       <Slider
-        aria-label="Sections"
+        aria-label="characters"
         defaultValue={0}
         marks={mark}
         value={sliderVal}
-        valueLabelDisplay="off"
+        valueLabelDisplay="on"
```

Comment:
I think having a similar label for the section slider number could be useful for finding interactions in a specific chapter in a book? Not an urgent thing though, LGTM other than that :)

Reply:
I'll have a look into this, although it may require making the chapter slider discrete again. I believe the slider is currently made to look continuous by multiplying the number of discreet steps by some large constant which means the value the slider shows would be off. I'll see if I can find a way around this :)